app/parse.py

```diff
@@ -1,7 +1,13 @@
+import requests
+from bs4 import BeautifulSoup
+
 from dataclasses import dataclass
 from enum import Enum
```

Comment:
Consider organizing your imports. Typically, it's recommended to group them like this: standard libraries first, followed by external libraries, and then your own modules. This improves code readability and maintainability